### PR TITLE
DRTII-1040 update south asia posts and countries

### PR DESCRIPTION
**Pull request by ngu04**

### CODE REVIEW

File: ui/src/components/RegionPortData.tsx [hunk 1]
```diff
@@ -193,7 +204,8 @@ export const newDelhiCountries = [
     {country: 'Afghanistan'},
     {country: 'Bhutan'},
     {country: 'Burma'},
-    {country: 'India'}
+    {country: 'India'},
```
Comment (richbirch) [inline ui/src/components/RegionPortData.tsx:207]: Maybe the ticket is ambiguous, but for me the ticket says New Delhi region should only have India and Nepal in it. Am I reading the ticket incorrrectly?

Comment (ngu04) [reply ui/src/components/RegionPortData.tsx:207]: As nothing mentioned about removing them I kept them as it is . Let me check with Rob about it .

File: ui/src/components/RegionPortData.tsx [hunk 2]
```diff
@@ -57,7 +55,10 @@ export const middleEastAndPakistanPost = [
 
 export const southAndSouthEastAsiaPost = [
     {post: 'All'},
+    {post: 'Bangkok'},
     {post: 'Hanoi'},
```
Comment (richbirch) [inline ui/src/components/RegionPortData.tsx:59]: Fix alphabetisation for South & South East Asia countries

Comment (richbirch) [reply ui/src/components/RegionPortData.tsx:59]: Istanbul countries also need re-ordering

**Review by richbirch — APPROVED** [review]
Good stuff. I just noticed a few countries out of order.

It would be really good to keep all the region/port/country definitions in scala and pass them through to the front end through an API call so that we only have them in one place. I'll add a ticket for that as it's unrelated to this

**Pull request merged**